dask/dataframe/core.py

```diff
@@ -3228,25 +3229,27 @@ def astype(self, dtype):
             M.astype, dtype=dtype, meta=meta, enforce_metadata=False
         )
 
-    @derived_from(pd.Series)
-    def append(self, other, interleave_partitions=False):
-        if PANDAS_GT_140:
-            warnings.warn(
-                "The frame.append method is deprecated and will be removed from"
-                "dask in a future version. Use dask.dataframe.concat instead.",
-                FutureWarning,
-            )
-        # because DataFrame.append will override the method,
-        # wrap by pd.Series.append docstring
-        from dask.dataframe.multi import concat
+    if not PANDAS_GT_200:
```

Comment:
It felt a little strange making this method conditional, but it seems to work and provides smooth compatibility across supported `pandas` versions. Open to other suggestions though